src/components/TeacherComponents/TeacherNotesModal/TeacherNotesModal.tsx

```diff
@@ -96,9 +96,9 @@ export const TeacherNotesModal = ({
   onClose,
   isOpen,
   teacherNote,
-  saveTeacherNote,
+  // saveTeacherNote,
 }: TeacherNotesModalProps) => {
-  const [noteSaved, setNoteSaved] = useState(false);
+  // const [noteSaved, setNoteSaved] = useState(false);
   const [noteShared] = useState(false);
 
```

Comment:
Simon has a PR to fix this, component not currently in use and was throwing type errors